src/linuxforhealth/csvtofhir/model/csv/basic.py

```diff
@@ -0,0 +1,28 @@
+from typing import List, Optional
+from pydantic import validator
+
+from linuxforhealth.csvtofhir.fhirutils import csv_record_validator as csvrecord
+from linuxforhealth.csvtofhir.fhirutils import fhir_utils
+from linuxforhealth.csvtofhir.fhirutils.fhir_constants import SystemConstants, ValueDataAbsentReason
+from linuxforhealth.csvtofhir.model.csv.base import CsvBaseModel
+from linuxforhealth.csvtofhir.model.csv.base import BaseModel
+
+
+class TokenDetails(BaseModel):
+    token_name: Optional[str]
+    token_value: Optional[str]
+    token_system: Optional[str]
+
+class BasicCsv(CsvBaseModel):
+    identifierTypeSystem: Optional[str]        # system for token identifiers
+    baseSystem: Optional[str]                  # base system
+    identifierNamePrefix: Optional[str]        # prefix for identifier ids
+    tokenList: Optional[List[str]]             # token identifiers
+    patientInternalIdentifier: Optional[str]   # id
+    # datavant
+    token_encryption_key: Optional[str]        # other identifier
+    tokenized_sid: Optional[str]               # other identifier
+    source_patient_sid: Optional[str]          # other identifier
+    created_date: Optional[str]                # subject
```

Comment:
correct comment

Reply:
Not sure if you mean the comment is correct or you are saying correct the comment. I believe this are all right so let me know.

Reply:
Let's dicuss when you get back

Reply:
I am referring to the comment "# subject" on created_date

Reply:
Fixed